libs/blocks/global-navigation/global-navigation.js

```diff
@@ -233,23 +223,28 @@ class Gnav {
 
   loadDelayed = async () => {
     this.ready = this.ready || new Promise(async (resolve) => {
-      this.el.removeEventListener('click', this.loadDelayed);
-      this.el.removeEventListener('keydown', this.loadDelayed);
-      const [
-        { appLauncher },
-        ProfileDropdown,
-        Search,
-      ] = await Promise.all([
-        loadBlock('../features/appLauncher/appLauncher.js'),
-        loadBlock('../features/profile/dropdown.js'),
-        loadBlock('../features/search/gnav-search.js'),
-        loadStyles('features/profile/dropdown.css'),
-        loadStyles('features/search/gnav-search.css'),
-      ]);
-      this.ProfileDropdown = ProfileDropdown;
-      this.appLauncher = appLauncher;
-      this.Search = Search;
-      resolve();
+      try {
+        this.el.removeEventListener('click', this.loadDelayed);
+        this.el.removeEventListener('keydown', this.loadDelayed);
+        const [
+          { appLauncher },
+          ProfileDropdown,
+          Search,
+        ] = await Promise.all([
+          loadBlock('../features/appLauncher/appLauncher.js'),
+          loadBlock('../features/profile/dropdown.js'),
+          loadBlock('../features/search/gnav-search.js'),
+          loadStyles('features/profile/dropdown.css'),
+          loadStyles('features/search/gnav-search.css'),
+        ]);
+        this.ProfileDropdown = ProfileDropdown;
+        this.appLauncher = appLauncher;
+        this.Search = Search;
+        resolve();
+      } catch (e) {
+        lanaLog({ message: 'Error within loadDelayed', e });
```

Comment:
This [line](https://github.com/adobecom/milo/blob/f3012c8045c629f83a2ed41d91c54761c6adf23a/libs/blocks/global-navigation/global-navigation.js#L274) has a message specific to 'Global Navigation'. I know that we have our own client ID, but since multiple platforms might have a `loadDelayed` method and there might be error spillover, I'd suggest we make this a bit more particular to 'Global Navigation' as well.